src/app/content/components/Content.browserspec.tsx

```diff
@@ -13,6 +14,14 @@ const EXPECTED_SCROLL_TOPS: { [testCase: string]: number[] } = {
   Mobile: [239, 66, 96, 239, 523, 1263, 1402, 1751, 2118],
 };
 
+beforeAll(async() => {
+  await page.setCookie({
```

Comment:
idk what change i made but the nudge became a bigger problem in some of these tests, the cookie makes it not obscure the rest of the test behavior